BLUESPAWN-win-client/headers/util/permissions/permissions.h

```diff
@@ -212,6 +274,70 @@ namespace Permissions{
 		 * @return The name of the owner
 		 */
 		virtual std::wstring ToString() const;
+
+		/**
+		* Function to get a list of privileges the owner has
+		*
+		* @return the list of privileges an owner explicity has, and if they 
+		*     are a user, includes the privileges granted to them by the
+		*     groups they're in.
+		*/
+		std::vector<std::wstring> GetPrivileges();
```

Comment:
it might be a good idea to eventually make something better to represent a privilege. Strings don't feel like a natural program representation. This is fine for now though

Reply:
Agreed